README.md

```diff
@@ -27,49 +28,61 @@
 > This action presumes you already have a zeroheight account. You can [sign up here](https://zeroheight.com/create/account?utm_department=marketing&utm_source=github).
 
 1. Head to [`zeroheight.com/adoption`](https://zeroheight.com/adoption) and follow the steps to connect your design system packages and application source code to zeroheight
-3. Follow the "Use the CLI" flow to create a Client ID and Access Token and ensure to note these down for future use
-4. Now head over to your application's source code on GitHub and enter your zeroheight Client ID and Access Token as GitHub repository secrets (Settings → Secrets and variables → Actions) using `ZEROHEIGHT_CLIENT_ID` and `ZEROHEIGHT_ACCESS_TOKEN` respectively
-5. Once completed you can use the action as part of your GitHub Action workflow file
-6. Once your workflow is updated, trigger your action, and check in zeroheight to see design system usage data come in 🎉
+2. Follow the "Use the CLI" flow to create a Client ID and Access Token and ensure to note these down for future use
+3. Now head over to your application's source code on GitHub and enter your zeroheight Client ID and Access Token as GitHub repository secrets (Settings → Secrets and variables → Actions) using `ZEROHEIGHT_CLIENT_ID` and `ZEROHEIGHT_ACCESS_TOKEN` respectively
+4. Once completed you can use the action as part of your GitHub Action workflow file
+5. Once your workflow is updated, trigger your action, and check in zeroheight to see design system usage data come in 🎉
```

Comment:
Do we want to include a link to adoption here?

Reply:
What part of adoption do you mean? As in straight into their own dashboard?